### PR TITLE
fix gpu workflow

### PR DESCRIPTION
**Pull request by harborn**

### CODE REVIEW

File: .github/workflows/workflow_finetune_gpu.yml [hunk 1]
```diff
@@ -38,9 +38,9 @@ jobs:
 
       - name: Running task on Intel GPU
         run: |
-          rm ~/borealis-runner/llm-on-ray.tar.gz -f
-          tar zcf ~/borealis-runner/llm-on-ray.tar.gz -C ~/actions-runner/_work/llm-on-ray .
-          cd ~/borealis-runner/
+          rm /home/ci/borealis-runner/llm-on-ray.tar.gz -f
+          tar zcf /home/ci/borealis-runner/llm-on-ray.tar.gz -C /home/ci/actions-runner/_work/llm-on-ray .
+          cd /home/ci/borealis-runner/
           python3 finetune_on_pvc.py --need_create_conda_env true --base_models "${{ matrix.model }}"
```
Comment (jiafuzha) [inline .github/workflows/workflow_finetune_gpu.yml:44]: finetune_on_pvc.py should be checked in to our repo.

Comment (harborn) [reply .github/workflows/workflow_finetune_gpu.yml:44]: finetune_on_pvc.py include some confidential information, may not add to our repo.

Comment (jiafuzha) [reply .github/workflows/workflow_finetune_gpu.yml:44]: please check in finetune_on_pvc.py to our internal project in case of disk failure.

**Review by jiafuzha — APPROVED** [review]
LGTM

**Pull request merged**